persistence-api/src/test/java/io/stargate/db/schema/ColumnTest.java

```diff
@@ -251,11 +253,11 @@ public void testToString() throws UnknownHostException {
   @Test
   public void testToStringWithCollections() {
     java.util.List<Integer> list = Arrays.asList(1, 2, 3);
-    Set<Integer> set = new HashSet<>(list);
+    java.util.TreeSet<Integer> set = new TreeSet(list);
+    String mapString = Type.Map.of(Type.Int, Type.Int).toString(ImmutableMap.of(1, 1, 2, 2));
     assertThat(Type.List.of(Type.Int).toString(list)).isEqualTo("[1,2,3]");
     assertThat(Type.Set.of(Type.Int).toString(set)).isEqualTo("{1,2,3}");
-    assertThat(Type.Map.of(Type.Int, Type.Int).toString(ImmutableMap.of(1, 1, 2, 2)))
-        .isEqualTo("{1:1,2:2}");
+    assertTrue((mapString).equals("{1:1,2:2}") || (mapString).equals("{2:2,1:1}"));
```

Comment:
Could we use `TreeMap` here?

```java
    Map<Integer, Integer> m = new TreeMap<>();
    m.put(1, 1);
    m.put(2, 2)
```

Reply:
I think a bot created this PR.. Can we finalize it ourselves?